scripts/postinstall.js

```diff
@@ -20,13 +20,14 @@ try {
 
 function setupAutocomplete() {
   return new Promise((resolve, reject) => {
+    const tabtabPath = require.resolve('tabtab').replace(/\/index.js$/, '');
```

Comment:
I think this is not OS agnostic (also commented by @hash) as it uses a fixed "/" path separator which is e.g. not available on Windows.
`path.sep` should be used to build the regex here.